seeds/4_publishedProjects.js

```diff
@@ -0,0 +1,15 @@
+exports.seed = function(knex, Promise) {
+  return Promise.join(
+    knex('publishedProjects').insert({
+      title: 'sinatra-contrib',
+      tags: 'ruby, sinatra, community, utilities',
+      description: 'Hydrogen atoms Sea of Tranquility are creatures of the cosmos shores of the cosmic ocean.',
+      date_created: '2015-06-03T16:21:58+00:00',
```

Comment:
This does not look like a proper simplified ISO time, which would be `2015-06-03T16:21:58.000Z'